src/main/objects/BassTranslator.js

```diff
@@ -348,6 +358,11 @@ function BassTranslator() {
             revolution++
         }
 
+        if (mode === Consts.MODE.MINOR && revolution === 3
```

Comment:
Tego fragmentu nie rozumiem...

Reply:
Nie trzeba w takiej sytuacji dać revolution 3> zamiast 3?

Reply:
To czy 3>, czy 3 to wynika ze skali. Mode mówi nam czy mamy użyć minor czy major, i obliczanie tercji to jest to o czym pisaliśmy w kwesti HS-76 :)

Reply:
to wyrzucę :)